rules/opentitan_test.bzl

```diff
@@ -320,6 +323,12 @@ def opentitan_functest(
         if signed:
             flash += "_signed_{}".format(key)
 
+        # Reference all outputs of the test_binary target, so they are built.
+        test_binary_data = []
+        if add_flash_side_outputs:
+            test_binary_filegroup = "{}_{}_filegroup".format(test_binary, target)
+            test_binary_data.append(test_binary_filegroup)
```

Comment:
completey dumb question, if all the binary stuff is included, can we simplify some of the `concat_arg` stuff that previously handled the `vmems`? Or is that unrelated.

Reply:
It's unrelated. _format_list() looks like it's being used to construct arguments sent to opentitantool (by filling in placeholders in the device-specific params).

By contrast, this filegroup merely makes available more files in bazel's sandbox for the sh_test, but it doesn't instruct opentitantool how to use them (since it doesn't use them). Because we tell bazel the files need to be available, though, it dutifully builds the related targets.

Reply:
Can we also cleanup the need to specify the SW logging database files for DV sim? This was added in above on line 317 since the final sh_test rule wasn't listing this as a direct dep.